data/flags/man_made/flagpole.json

```diff
@@ -3432,6 +3432,30 @@
         "subject:wikidata": "Q804"
       }
     },
+    {
+      "displayName": "Pan-African",
+      "id": "panafrican-e5dc93",
+      "locationSet": {"include": ["001"]},
+      "matchNames": [
+        "afro-american",
+        "bendera ya taifa",
+        "black liberation",
+        "international african",
+        "marcus garvey",
+        "rbg",
+        "red black green",
+        "unia",
+        "universal african"
```

Comment:
These names came from [this Wikipedia article](https://en.wikipedia.org/wiki/Special:PermanentLink/1091075171#Alternative_names).